app/views/insured/families/_enrollment_actions.html.erb

```diff
@@ -1,61 +1,61 @@
 <% can_make_changes_for_enrollment = hbx_enrollment.can_make_changes? %>
 
 <div class="enrollment-actions-ddl">
-    <% unless read_only || !can_make_changes_for_enrollment %>
-        <div class="dropdown">
-            <button class="btn btn-default dropdown-toggle interaction-click-control-actions" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                <%= l10n("actions")%>
-                <span class="caret"></span>
-            </button>
+  <% unless read_only || !can_make_changes_for_enrollment %>
+    <div class="dropdown">
+      <button class="btn btn-default dropdown-toggle interaction-click-control-actions button outline " type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <%= l10n("actions")%>
+        <span class="caret"></span>
+      </button>
 
-            <div class="dropdown-menu dropdown-glossary run-glossary" role="menu" aria-labelledby="dropdownmenuButton">
-                <%# coverage details %>
-                <div class="dropdown-divider" style="border-top: 1px solid #e9ecef"></div>
-                <a onfocus="handleGlossaryFocus('view_coverage_details')" onblur="handleGlossaryBlur('view_coverage_details')" href="<%= summary_products_plans_path({standard_component_id: hbx_enrollment.product.hios_id, hbx_enrollment_id: hbx_enrollment.id, active_year: hbx_enrollment.product.active_year, source: 'account'}) %>" class='btn-link btn-block dropdown-item' id='view-details-btn' style='padding: 6px 12px; margin: 4px 0;' data-remote='true'>
-                    <%= render partial:"shared/glossary_hover", locals: {key: "view_coverage_details", title: "View my coverage details", term: sanitize(l10n("view_details")) } %>
-                </a>
+      <div class="dropdown-menu dropdown-glossary run-glossary" role="menu" aria-labelledby="dropdownmenuButton">
+        <%# coverage details %>
+        <div class="dropdown-divider" style="border-top: 1px solid #e9ecef"></div>
+        <a onfocus="handleGlossaryFocus('view_coverage_details')" onblur="handleGlossaryBlur('view_coverage_details')" href="<%= summary_products_plans_path({standard_component_id: hbx_enrollment.product.hios_id, hbx_enrollment_id: hbx_enrollment.id, active_year: hbx_enrollment.product.active_year, source: 'account'}) %>" class='btn-link btn-block dropdown-item' id='view-details-btn' style='padding: 6px 12px; margin: 4px 0;' data-remote='true'>
+          <%= render partial:"shared/glossary_hover", locals: {key: "view_coverage_details", title: "View my coverage details", term: sanitize(l10n("view_details")) } %>
```

Comment:
should glossary_hover be updated for bs4? also, I know `shared/glossary` already is uplifted for bs4, can we use that instead?